public/i18n/en.json

```diff
@@ -36,6 +36,7 @@
     "log-in-message": "Please <a href=\"$1\">log in</a> to be able to make any changes.",
     "results-page-description": "You can fix them! The table below presents you with links to the statement on Wikidata and to the entry on the external source. You can compare the values and manually fix the mismatch, editing Wikidata or the external data source. After investigating and fixing the mismatch, you can indicate their status using the options from the drop-down.",
     "results-page-title": "What should I do with the mismatches?",
+    "results-instructions-button": "Instructions",
```

Comment:
I guess then this has to be changed to "Instruction".. :shrug:

Reply:
Changed. :man_shrugging: 

Isn't it funny that I use "instruction**s** button" and "instruction**s** dialog" everywhere in the code? :-)

Reply:
Yes, I am almost sure there will be copy changes here.